java-frontend/src/test/java/org/sonar/java/viewer/EGViewer.java

```diff
@@ -42,6 +42,7 @@
 
   private static final ActionParser<Tree> PARSER = JavaParser.createParser(StandardCharsets.UTF_8);
   private final Viewer viewer;
+  private static final boolean SHOW_CACHE = true;
```

Comment:
This is an implementation details that I want to see appearing in the EG Viewer, which is a debugging tool. Knowing that a link come from the cache or not seems then important to me.